compiler/include/expr.h

```diff
@@ -222,6 +227,41 @@ class CallExpr : public Expr {
   bool            isPrimitive(const char*  primitiveName)                const;
 };
 
+// For storing several call expressions, where
+// choosing between them depends on context
+// (and that choice might need to be done later in resolution).
+// These should only exist during resolution.
```

Comment:
I was thinking of resolution as including cullOverReferences. I'll update the comment to be more exact.